app/helpers/manage_metadata_helper.rb

```diff
@@ -33,6 +33,8 @@ def current_manage_title
       'manage_variables'
     elsif is_services_controller?
       'manage_services'
+    elsif controller.lookup_context.prefixes.include?('proposal/manage_collection_proposals')
```

Comment:
I feel like this can be even more generic. For the time being anything namespaced under `proposal` should give this title. Also, does this miss everything in CollectionDraftProposalsController? seems to me like it would

Reply:
Now it looks for an entry with prefix 'proposal'